cat-theme/src/css/abstracts/_mixins.scss

```diff
@@ -0,0 +1,27 @@
+@mixin flex($flex-direction: row, $justify-content: center, $align-items: center) {
+    display:flex;
+    flex-direction: $flex-direction;
+    justify-content: $justify-content;
+    align-items: $align-items;
+}
+
+/* ---------- Media Queries ---------- */
+
+@mixin respond($breakpoint) {
+    @if $breakpoint == phone {
+        @media only screen and (max-width: 37.5em) { @content };    //600px
```

Comment:
For an inline comment, we only need 1 space at the end of the `;` and the start of the `//`.